src/com/jalasoft/search/view/SearchPanelCriteria.java

```diff
@@ -81,9 +81,10 @@
     private JDateChooser fromAccessedDate;
```

Comment:
Add comments for  getter methods in SearchPanelCriteria.Java, (these methods in other files have a short documentation)